packages/backend/src/models/OrganizationMemberProfileModel.ts

```diff
@@ -148,9 +157,21 @@ export class OrganizationMemberProfileModel {
             query,
             paginateArgs,
         );
+        const usersHaveAuthenticationRows =
+            await UserModel.findIfUsersHaveAuthentication(this.database, {
+                userUuids: data.map((m) => m.user_uuid),
+            });
+
         return {
             pagination,
-            data: data.map(OrganizationMemberProfileModel.parseRow),
+            data: data.map((m) =>
+                OrganizationMemberProfileModel.parseRow(
+                    m,
+                    usersHaveAuthenticationRows.find(
```

Comment:
Not optimal to have a loop within a loop, could this be broken out to something like:

```
const usersHaveAuthenticationMap = new Map(
        usersHaveAuthenticationRows.map(row => [row.user_uuid, row.has_authentication])
    );

    return {
        pagination,
        data: data.map((m) =>
            OrganizationMemberProfileModel.parseRow(
                m,
                usersHaveAuthenticationMap.get(m.user_uuid) || false,
            ),
        ),
    };
```

Reply:
Nice one :) refactored the code with that 👍